packages/core/test/core.test.ts

```diff
@@ -19,38 +19,38 @@ describe('PCD tests', function () {
 
   it('PCD flow location prover', async function () {
     const artifactsDirName = __dirname + '/../../circuits/artifacts'
```

Comment:
Should we set path to aadhaar-verifier is constant, and export it in core circuit packages. 
This help developer can import it more easier.